tardis/energy_input/GXPacket.py

```diff
@@ -79,6 +79,32 @@ def get_location_r(self):
         )
 
 
+class GXPacketCollection:
+    def __init__(
+        self,
+        location,
+        direction,
+        energy_rf,
+        energy_cmf,
+        nu_rf,
+        nu_cmf,
+        status,
+        shell,
+        time_current,
+    ):
+        self.location = location
+        self.direction = direction
+        self.energy_rf = energy_rf
+        self.energy_cmf = energy_cmf
+        self.nu_rf = nu_rf
+        self.nu_cmf = nu_cmf
+        self.status = status
+        self.shell = shell
+        self.time_current = time_current
+        self.number_of_packets = len(self.energy_rf)
+        self.tau = -np.log(np.random.random(self.number_of_packets))
```

Comment:
Is this `tau` same as the mean life of isotopes? Shall we give a separate name for this variable if this the optical depth?

Reply:
Given that we are in a radiative transfer code, I think we should rename the mean lifetime instead